lib/internal/modules/package_json_reader.js

```diff
@@ -38,4 +38,14 @@ function read(jsonPath) {
   return result;
 }
 
-module.exports = { read };
+/**
+ * 
+ * @param {string} jsonPath 
+ * @param {string} srcPath 
+ */
+function clobber(jsonPath, srcPath) {
+  const value = cache.has(srcPath) ? cache.get(srcPath) : read(srcPath);
```

Comment:
Could this just be `const value = cache.get(srcPath) || read(srcPath)` ?

Reply:
since we never put a falsey value in currently, yes it could. I tend to just use `.has` as it avoids me needing to worry about changes like potentially introducing a falsey value